flank-scripts/src/main/kotlin/flank/scripts/ops/integrationtests/common/ITResult.kt

```diff
@@ -1,5 +1,20 @@
 package flank.scripts.ops.integrationtests.common
 
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+@Serializable(with = ITResultSerializer::class)
 enum class ITResult {
     SUCCESS, FAILURE
 }
+
+object ITResultSerializer : KSerializer<ITResult> {
```

Comment:
Are we sure this is needed? 
@SerialName should do the trick?

Reply:
It would, my intention was that scripts would handle both `SUCCESS` and `success`, not only lower cases

Reply:
I am shocked they have yet to implement a type of alias as apart of the @SerialName parameter.
https://github.com/Kotlin/kotlinx.serialization/issues/203
is the only info they have

Reply:
Yeah, and I can see that related PR is stuck :/